### DISCUSSION TITLE
TabView binds

### DISCUSSION THREAD

**Discussion by CezaryPuchowski** [TomSchimansky/CustomTkinter#1284] [Q&A] [answered]
Hi,
I would like bind callback function to CTkTabview Tab Changed event, but I can't find information about implemented sequences.
`tabview.bind('<<NotebookTabChanged>>', command = tabClick)` returns `NotImplementedError:`

Can someone help?

**Comment by Wolf-SO** [comment, marked as answer]
This is not possible via `bind` because `CTkTabview` is not just an adapter for a native Tk widget like [ttk::notebook](https://www.tcl.tk/man/tcl8.7/TkCmd/ttk_notebook.html). As I dead the documentation, [the `command` parameter](https://github.com/TomSchimansky/CustomTkinter/wiki/CTkTabview) seems the best option to me,

**Reply by CezaryPuchowski** [reply]
Yes, thank You.
In function called by pressing any tab I can now run `tabview.get()` to get name of tab I pressed. :)

**Reply by MaxNiark** [reply]
Hello Cezary, 

I'm trying to get the tab name when i click on but actually i get only the first call of the widget CtkFrame from tabview :( ... 

Could you share your code ? :) 

thanks !

**Reply by CezaryPuchowski** [reply]
Hi,
Yes, I experience the same thing. If you want call every click for specific tab, you can check code below:
```python
import customtkinter as ctk

# create main window, size 500x400px and 20,20 px margin from top left corner
app = ctk.CTk()
app.geometry("500x400+20+20")

# tabView tabs names
tab1Name = "Tab1Name"
tab2Name = "Tab2Name"

def tabViewTabChangeClick():
    print(f"Click event once on new clicked tab. You Click: {tabView.get()}")

def tabViewTab1Click(event):
    print(f"Every click on tab1 body. You Click: {event}")

tabView = ctk.CTkTabview(master=app, width=200, height=70, command=tabViewTabChangeClick)
tabView.pack(expand=True, fill="both")

tab1 = tabView.add(tab1Name)
tab2 = tabView.add(tab2Name)

tab1.bind("<Button-1>", command=tabViewTab1Click)

app.mainloop()
```

**Reply by MaxNiark** [reply]
i love you :)
